### DISCUSSION TITLE
what can i do to get this simple api call working?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#131] [Q&A] [answered]
Hello Renaud,

It is just an http endpoint to api.chucknorris.io/jokes/random and I can not get it to work. It used to work a few weeks ago.
What is going wrong?

nb im just practising with simple api's but later on i want to get my appointments of today from Google Calender API to fuel a select.

Kind regards

Folkert

{
  "cid": "index",
  "type": "ContainerView",
  "dataType": "object",
  "layout": "block",
  "components": [
    {
      "cid": "container-0",
      "type": "ContainerView",
      "dataType": "object",
      "layout": "block",
      "components": [
        {
          "cid": "http-1",
          "type": "HttpConnector",
          "dataType": "object",
          "method": "GET",
          "eventHandlers": [],
          "baseUrl": "https://api.chucknorris.io/jokes/random"
        },
        {
          "cid": "button-1",
          "type": "ButtonView",
          "dataSource": "$parent",
          "label": "Click me",
          "buttonType": "button",
          "eventHandlers": [
            {
              "global": false,
              "name": "@click",
              "actions": [
                {
                  "targetId": "http-1",
                  "name": "update",
                  "description": "Default action"
                }
              ]
            }
          ]
        },
        {
          "cid": "textarea-0",
          "type": "TextareaView",
          "dataType": "string",
          "label": "",
          "description": "",
          "field": "",
          "size": "default",
          "disabled": false,
          "placeholder": "",
          "eventHandlers": [],
          "dataSource": "http-1"
        }
      ],
      "defaultValue": "={}",
      "dataSource": "$parent",
      "eventHandlers": []
    }
  ],
  "defaultValue": "={}",
  "eventHandlers": []
}

**Comment by renaudpawlak** [comment, marked as answer]
Oops, sorry there was a bug that I noticed today. I just made a new release with the fix. Should be working again now :)

**Comment by HFolkertH** [comment]
Top, it works again
